csharp/ExcelAddInInstaller/CustomActions/README.md

```diff
@@ -0,0 +1,121 @@
+# Background
+
+The purpose of this library is to add a "Custom Action" to our
+Advanced Installer package. This custom action does the actions
+needed to manipulate the Windows Registry in order to do things like
+
+1. detect whether the version of Office installed is 32 or 64 bit
+2. Add the special keys that tell Excel to open an Excel Add-In at startup
+
+# Information about this library
+
+This library is a .NET 4.8.0 Class Library with some special boilerplate
+code provided by Advanced Installer.
+
+.NET 4.8.0 is pretty old at this point, but I chose it because (I believe)
+it is guaranteed to be present on Windows 10/11 installations. Note that
+this is *not* the runtime used by the Excel Add-In itself; that add-in uses
+a much more modern runtime (.NET 8). This is just the runtime used to
+support the custom actions (registry manipulations) in the installer.
+
+This library and its boilerplate code were created by adding a
+Visual Studio Extension provided by Advanced Installer to Visual Studio.
+The process for adding the extension is documented here:
+
+https://www.advancedinstaller.com/user-guide/create-dot-net-ca.html
+
+Basically the steps are:
+
+* Open Visual Studio and navigate to Extensions → Manage Extensions.
+* In the Online section, search for Advanced Installer for Visual Studio
+* In Visual Studio navigate to File → New Project
+* From the list of templates, select the C# Custom Action template or the
+  C# Custom Action (.NET Framework) template, depending on your needs
+
+Because of the above compatibility requirements I have decided that
```

Comment:
I don't understand this paragraph.

Reply:
Mangled because it was a mixture of copy-pasted text (from the Advanced Installer manual) with my own text.

PTAL at my attempt at explaining this better.